lisa/util/__init__.py

```diff
@@ -750,3 +753,25 @@ def check_till_timeout(
         sleep(interval)
     if timer.elapsed() >= timeout:
         raise LisaException(f"timeout: {timeout_message}")
+
+
+def retry_with_excluded_exceptions(
```

Comment:
The "excluded" and "skip" in the parameters should be consistent.

Reply:
updated